app/mailers/visitor_mailer.rb

```diff
@@ -73,13 +67,11 @@ def first_date
     Date.parse(@visit.slots.first.date)
   end
 
-  def receipt_date
-    first_date.strftime('%e %B %Y').gsub(/^ /,'')
+  def slot_date
+    Date.parse(@slot.date)
   end
 
-  alias_method :rejection_date, :receipt_date
-
-  def confirmation_date
-    Date.parse(@slot.date).strftime('%e %B %Y').gsub(/^ /,'')
+  def format_date(date)
+    date.strftime('%e %B %Y').gsub(/^ /,'')
```

Comment:
It's also worth looking into why this date has a year and the one above doesn't. Perhaps they could both be the same.